_includes/code/howto/manage-data.import.py

```diff
@@ -49,6 +49,13 @@
             properties=data_row,
         )
 # highlight-end
+        if batch.number_errors > 0:
+            print(f"Error found during batch import")
```

Comment:
Hey @g-despot, this is not a good idea 😉
Here is why.
Imagine you are importing 1 million objects.
When you get to around 20k objects imported, you get an error. Now, the batch loop doesn't stop, it continues on. 
And because the number_errors will always be > 0, you will get "Error found during batch import" printed 980k times 🫣

`batch.number_errors` is useful to stop the batch loop if you get too many errors, like this:
```
if(batch.number_errors > 10):
  print("Too many errors during batch import. Stopping the import.")
  break
```

Reply:
Thanks for noticing this, it's definitely crucial to exit the loop in this scenario. 🥲